roles/edpm_nftables/tasks/service-bootstrap.yml

```diff
@@ -31,6 +31,9 @@
       loop:
         - iptables.service
         - ip6tables.service
+      when:
+        - ansible_facts.services["iptables.service"] is defined
+        - ansible_facts.services["ip6tables.service"] is defined
```

Comment:
This should probably be an OR to maintain the previous behavor where either service may or may not exist.

Reply:
`iptables` and `ip6tables` service aren't installed together?

Reply:
https://access.redhat.com/solutions/1411953

in EL9:

- `iptables-nft-services` provides both services (v4 and v6)
- isn't installed by default
- it's deprecated

I can change it the condition to be an or but if one of the services isn't present it's not the standard scenario.